src/praisonai/tests/unit/cli/test_config_single_source.py

```
@@ -0,0 +1,269 @@
+                    "servers": {
+                        "playwright": {"command": ["npx", "-y", "@playwright/mcp"]}
+                    }
+                },
+                "permissions": {
+                    "default": "ask",
+                    "rules": [{"pattern": "bash:rm *", "action": "deny"}],
+                },
+            },
+        )
+
+        mcp, mcp_env, perms = self._resolve_in_dir(
```

Ruff (0.15.18)

[warning] Unpacked variable `mcp_env` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)